internal/server/statvar/fetcher/formula_fetcher.go

```diff
@@ -39,51 +41,70 @@ func FetchFormulas(
 	remoteRespChan := make(chan *pbv2.NodeResponse, 1)
 	// Fetch for BT and SQL.
 	errGroup.Go(func() error {
-		data, _, err := v1pv.Fetch(
-			errCtx,
-			store,
-			[]string{"StatisticalCalculation"},
-			[]string{"typeOf"},
-			0,
-			"",
-			"in",
-		)
-		if err != nil {
-			return err
-		}
-		statCal := []string{}
-		for _, nodes := range data["StatisticalCalculation"]["typeOf"] {
-			for _, node := range nodes {
-				statCal = append(statCal, node.Dcid)
+		statCalDcids := []string{}
+		nextToken := ""
+		for {
+			statCalReq := &pbv1.TriplesRequest{
+				Node:      "StatisticalCalculation",
+				Direction: "in",
+				NextToken: nextToken,
+			}
+			statCalResp, err := triples.Triples(
+				errCtx,
+				statCalReq,
+				store,
+				metadata,
+			)
+			if err != nil {
+				return err
+			}
+			typeOf, ok := statCalResp.Triples["typeOf"]
+			if ok {
+				for _, node := range typeOf.Nodes {
+					statCalDcids = append(statCalDcids, node.Dcid)
+				}
+			}
+			nextToken = statCalResp.GetNextToken()
+			if nextToken == "" {
+				break
 			}
 		}
-		if len(statCal) == 0 {
+		if len(statCalDcids) == 0 {
 			return nil
 		}
-		resp, _, err := v1pv.Fetch(
-			errCtx,
-			store,
-			statCal,
-			[]string{"outputProperty", "inputPropertyExpression"},
-			0,
-			"",
-			"out",
-		)
-		if err != nil {
-			return err
-		}
-		// Wrap result in pbv2.NodeResponse.
 		localResp := &pbv2.NodeResponse{Data: map[string]*pbv2.LinkedGraph{}}
-		for dcid, data := range resp {
-			localResp.Data[dcid] = &pbv2.LinkedGraph{
-				Arcs: map[string]*pbv2.Nodes{},
+		for {
+			bulkReq := &pbv1.BulkTriplesRequest{
+				Nodes:     statCalDcids,
+				Direction: "out",
+				NextToken: nextToken,
+			}
+			bulkResp, err := triples.BulkTriples(
+				errCtx,
+				bulkReq,
+				store,
+				metadata,
+			)
+			if err != nil {
+				return err
 			}
-			for prop, nodes := range data {
-				if len(nodes) > 0 {
-					localResp.Data[dcid].Arcs[prop] = &pbv2.Nodes{
-						Nodes: v1pv.MergeTypedNodes(nodes),
-					}
+			// Wrap result in pbv2.NodeResponse.
+			for _, nodeTriples := range bulkResp.Data {
+				out, outOk := nodeTriples.Triples["outputProperty"]
```

Comment:
nit: Consider making the string literals that are used in multiple places constants.

Reply:
Good idea, done